### PR TITLE
Branch support

### PR DESCRIPTION
**Pull request by DreierF**
Hey,

As already stated in #8 I would find it very convenient if the crowdin-plugin would support [branches](https://support.crowdin.com/versions-management/).
If you have any questions or considerations let me know 🙂

Thanks in advance

### CODE REVIEW

**Comment by mendhak** [comment]
Hey thanks, I don't get much time to work on this so always happy to look at pull requests.

Just to confirm, not specifying a branch will still work, yes?  So in other words,  the branches are optional for users

File: plugin/src/main/groovy/com/mendhak/gradlecrowdin/DownloadTranslationsTask.groovy [hunk 1]
```diff
@@ -10,40 +10,49 @@ class DownloadTranslationsTask extends DefaultTask {
...
-        def buildSubDir = new File(project.buildDir.getPath() , "/crowdin-plugin")
+        def buildSubDir = new File(project.buildDir.getPath(), "/crowdin-plugin")
         buildSubDir.mkdirs()
 
         //Tell Crowdin to do an export
-        def exportUrl = sprintf('http://api.crowdin.net/api/project/%s/export?key=%s', [projectId, apiKey])
-        ant.get(src: exportUrl, dest: new File(buildSubDir.getPath(), "export.xml"), verbose:true)
+        def exportUrl = sprintf('https://api.crowdin.com/api/project/%s/export?key=%s', [projectId, apiKey])
+        if (branch != null) {
+            exportUrl += '&branch=' + branchEncoded
```
Comment (mendhak) [inline plugin/src/main/groovy/com/mendhak/gradlecrowdin/DownloadTranslationsTask.groovy:25]: OK just figured out why you're doing this.  You can't just leave `&branch=`, their API doesn't handle empty values here, how annoying!

**Comment by mendhak** [comment]
OK! I have merged and released. Since this is a big enough feature I made it `0.1.0` :grin: 

It's on the [Gradle plugin portal](https://plugins.gradle.org/plugin/com.mendhak.gradlecrowdin) - can you pull this version and test yourself.  If any changes needed we can release more no problem. I did test myself and my non-branch features work just fine. 

It's on [Bintray](https://bintray.com/mendhak/maven/gradle-crowdin-plugin/)

And it'll be on [mvnrepository](https://mvnrepository.com/artifact/com.mendhak.gradlecrowdin/crowdin-plugin) in the next 5-10 years, seeing the glacial speed at which their repositories update...

**Comment by DreierF** [comment]
Thank you so much!

**Pull request merged**